### PR TITLE
small fixes

### PR DESCRIPTION
**Pull request by vafl**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/gluonts/model/npts/_model.py [hunk 1]
```diff
@@ -168,12 +168,11 @@ def predict(
...
 
+        freq = targets.index.freq.freqstr
         forecast_start = targets.index[-1] + 1 * targets.index.freq
 
         return SampleForecast(
-            samples=samples_pred_range,
-            start_date=forecast_start,
-            freq=forecast_start.freqstr,
+            samples=samples_pred_range, start_date=forecast_start, freq=freq
         )
```
Comment (timoschowski) [inline src/gluonts/model/npts/_model.py]: I don't understand why this change is necessary.

**Comment by szha** [comment]
Job PR-361/1 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-361/1/index.html

File: src/gluonts/model/npts/_model.py (hunk 1 above)
Comment (vafl) [reply src/gluonts/model/npts/_model.py]: In pandas versions older than 0.25 the frequency is not carried along when the time stamp is shifted.

**Comment by szha** [comment]
Job PR-361/2 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-361/2/index.html
